src/lib.rs

```diff
@@ -47,19 +63,49 @@ pub struct SensorData {
     pub gyro: AxisData,
 }
 
+/// The `Led` struct represents the 3 LEDs on navigator board.
+///
+/// Properties:
+///
+/// * `first`: A Pin struct representing the Green LED pin.
```

Comment:
That's not how you document properties.
https://doc.rust-lang.org/rust-by-example/meta/doc.html

Reply:
kept just the oe_pin, as inner comment for developers.